README.md

```diff
@@ -69,8 +69,6 @@ We keep here a limited list of canonical examples maintained by the Flax team. I
 ### Image Classification
 ⟶ [MNIST](examples/mnist) (also see [annotated version](https://flax.readthedocs.io/en/latest/annotated_mnist.html))
 
-⟶ [CIFAR-10](examples/cifar10) (Wide ResNet w/ and w/o Shake-Shake, PyramidNet w/ShakeDrop)
```

Comment:
Should we add it to "Flax examples from the community" and link to the google_research repository?

Reply:
It's already there:
https://github.com/google/flax/blame/master/examples/README.md#L37